Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -47,7 +47,11 @@
   </ItemGroup>
   
   <ItemGroup>
+    <Page Remove="Themes\League.Designer.xaml" />
+    <Page Remove="Themes\League.xaml" />
     <Page Remove="Themes\ThemeBuilder\Template.xaml" />
+    <Page Remove="Themes\Win11Dark.Designer.xaml" />
+    <Page Remove="Themes\Win11Dark.xaml" />
```

Comment:
Will it be considered as page? Why previous theme don't need to be remove here?

Reply:
I didn't know there was something like this. I'm going to erase it.